src/applications/yellow-ribbon/manifest.json

```diff
@@ -6,8 +6,8 @@
   "production": false,
   "template": {
     "vagovprod": false,
-    "vagovstaging": false,
-    "vagovdev": false,
+    "vagovstaging": true,
+    "vagovdev": true,
```

Comment:
You can just delete these lines instead of flipping the boolean. It defaults to `true`.